cms/server/admin/handlers/subtask_validators.py

```
@@ -0,0 +1,919 @@
+        if not success:
+            self.redirect(fallback_page)
+            return
+
+        try:
+            source_digest = self.service.file_cacher.put_file_content(
+                body, "Subtask validator source for %s" % task.name)
+            executable_digest = None
+            if compiled_bytes is not None:
+                executable_digest = self.service.file_cacher.put_file_content(
+                    compiled_bytes, "Subtask validator executable for %s" % task.name)
+        except Exception as error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)